modules/bookCollection.js

```diff
@@ -1,12 +1,6 @@
-class Book {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-  }
-}
-
-class BookCollection {
-  constructor() {
+/* eslint-disable */
```

Comment:
- [ ] As mentioned by the previous reviewer and for best practices and clean code, kindly ensure that you fix your linters errors instead of disabling them manually from your codebase.

Reply:
You are checking wrong I don't have any module with that name